docker/Dockerfile.fluxctl

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.3
```

Comment:
I don't think this needs any of the things in alpine, since it's a static binary. It's a static binary, right?